competitionapp/views.py

```diff
@@ -0,0 +1,53 @@
+from django.shortcuts import render
+from bs4 import BeautifulSoup
+import requests
+import re
+
+
+# Create your views here.
+def current_competitions_view(request):
```

Comment:
Nice function. 
Remarks:
- the if condition seems not needed
- I would put the parser into celery task and run daily or so, store result in DB and return from local database. The problem it would solve is when "https://zawodykonne.com/zawody/aktualne" is down, you stil lhave a local copy to work on.